pkg/apps/gitops.go

```diff
@@ -166,9 +176,18 @@ func (o *GitOpsOptions) DeleteApp(app string, alias string, autoMerge bool) erro
 
 // GetApps retrieves all the apps information for the given appNames from the repository and / or the CRD API
 func (o *GitOpsOptions) GetApps(appNames map[string]bool, expandFn func([]string) (*v1.AppList, error)) (*v1.AppList, error) {
-	dir, _, _, _, err := gits.ForkAndPullRepo(o.DevEnv.Spec.Source.URL, o.EnvironmentsDir, o.DevEnv.Spec.Source.Ref, "master", o.GitProvider, o.Gitter, "")
+	dir, err := ioutil.TempDir("", "create-pr")
```

Comment:
on a side note, I think we should start using a base dir within JX_HOME, e.g. JX_HOME/tmp for all out temporary directory creations. This probably makes it easier to debug and it is also easier to see when things don't get cleaned up.

Reply:
and why would 'get app' create a tmp directory with 'create-pr' in it?

Reply:
So for each 'get apps' we clone the repo? That does not seem terribly effective? I guess the idea of using ~/.jx/environment was to have some sort of a cache. 

Admittedly something which works it better than something which is "more performant" but not working ;-)

Reply:
`jx get app` and `jx get delete app` create PRs, so...

And yeah, I decided to ditch the sorta cache in `~/.jx/environments` because (a) it was horribly broken as written, and (b) there currently is no viable way to namespace the clone inside `~/.jx/environments` by cluster, so if you've got multiple clusters, you'd still end up having potential problems.

Reply:
wait, my brain fried - this shouldn't be `create-pr-`. duh. =)